sdk/src/main/java/com/kin/ecosystem/history/view/OrderHistoryRecyclerAdapter.java

```diff
@@ -125,46 +125,47 @@ private void setAmount(Order item) {
         }
 
         private void setSubtitle(Order item) {
-            StringBuilder subTitle = new StringBuilder();
+            String subTitle = "";
```

Comment:
why removed the string builder?

Reply:
it's separated now for the description view and date view